Src/DependencyCollector/Shared/Implementation/ProfilerSqlProcessingBase.cs

```diff
@@ -92,6 +92,24 @@ public object OnEndForOneParameter(object context, object returnValue, object th
         }
 
         /// <summary>
+        /// On end async callback for methods with 1 parameter.
+        /// </summary>
+        public object OnEndAsyncForOneParameter(object context, object returnValue, object thisObj)
```

Comment:
If I understand correctly the code uses different parameter signature versions depending on the number of parameters so the appropriate end handler is wired up.